### ISSUE TITLE
Build success and failure indicators don't contrast enough (accessibility issue)

### ISSUE THREAD

**Issue by jakepetroules** [travis-ci/travis-ci#754]
The green and red success and failure indicators on the new Travis design don't contrast well enough from each other. This presents an accessibility issue for persons with red-green color blindness.

Might I suggest that the red be made brighter so that they are easier to tell apart?
